### DISCUSSION TITLE
I didn't mean to create a new discussion, I'm unsure on how to delete this.

### DISCUSSION THREAD

**Discussion by SamGitHub1807** [sapondanaisriwan/AdashimaaTube#11] [Ideas]
I'm quite new to AdashimaaTube so I wasn't aware that I was supposed to comment under the discussion made by "sapondanaisriwan." I ended up creating a whole new discussion, it was entirely my fault, I should've checked and read it first. I apologize!

**Comment by sapondanaisriwan** [comment]
You don't have to apologize. It's not your fault. You can request the feature in the discussion section.

**Reply by SamGitHub1807** [reply]
Noted on this one. Thank you!
